editor-components/modal/confirmation-modal-1/confirmation.tsx

```diff
@@ -0,0 +1,142 @@
+            type: "image",
+            key: "image",
+            displayer: "image",
+            value: "https://picsum.photos/200"
+        });
+
+        this.addProp({
+            type: "icon",
+            key: "close-icon",
+            displayer: "icon",
+            value: "AiOutlineClose"
+        })
+
+        this.addProp({
+            type: "string",
+            key: "title",
+            displayer: "Title",
+            value: "Hang On a Sec!"
+        })
+
+        this.addProp({
+            type: "string",
+            key: "description",
+            displayer: "Description",
+            value: "Ready to rock and roll? Confirm your choice, and let's make some magic happen!"
+        })
+
+        this.addProp({
+            type: "object",
+            key: "button-1",
+            displayer: "Button",
+            value: [
+                {
+                    type: "string",
+                    key: "button1-text",
+                    displayer: "Button Text",
+                    value: "Count Me In"
+                },
+                {
+                    type: "page",
+                    key: "button-path",
+                    displayer: "Button Path",
+                    value: ""
+                }
+            ]
+        })
+
+        this.addProp({
+            type: "object",
+            key: "button-2",
+            displayer: "Button 2",
+            value:
+                    [{
+                        type: "string",
+                        key: "button2-text",
+                        displayer: "Button Text",
+                        value: "Let Me Rethink"
+                    }]
+             
+        })
+    }
+
+
+
+    getName(): string {
+        return "Confirmation Modal";
+    }
+
+    render() {
+
+        const closeIcon = this.getPropValue("close-icon");
+
+        return (
+            <div className={this.decorateCSS("container")}>
+                <div className={this.decorateCSS("max-content")}>
+                    <div className={this.decorateCSS("box")}>                 
+                        <div className={this.decorateCSS("head-content")}>
+                            {this.getPropValue("image") && (
+                                <img 
+                                    alt= ""
+                                    className={this.decorateCSS("image")}
+                                    src={this.getPropValue("image")}                        
+                                ></img>
+                            )}
+                            <button className={this.decorateCSS("close-button")}>
+                                <ComposerIcon name={closeIcon} propsIcon={{ size: 18, color: 'pink' }} />
+                            </button>
+                        </div>
+                        <div className={this.decorateCSS("bottom-content")}>
+                            {this.getPropValue("title") && this.getPropValue("title") !== "" && (
```

Comment:
assign this.getPropValue("title", { as_string : true}) to a variable. you will use it a few times. variables are more cleaner
```suggestion
                            {titleExist && (
```